packages/react-scripts/template/README.md

```diff
@@ -858,6 +859,44 @@ Snapshot testing is a new feature of Jest that automatically generates text snap
 
 This feature is experimental and still [has major usage issues](https://github.com/facebookincubator/create-react-app/issues/372) so we only encourage you to use it if you like experimental technology. We intend to gradually improve it over time and eventually offer it as the default solution for testing React components, but this will take time. [Read more about snapshot testing.](http://facebook.github.io/jest/blog/2016/07/27/jest-14.html)
 
+## Turbo Charge Component Development with React Storybook
+
+Usually, in our app, we have a lot of UI components, and they've different states. 
+For an example, a simple button component could have following states:
+
+* With a text label
+* With some emoji
+* With disabled mode
+
+With React Storybook, you can manage these different states (or stories) and develop them without running your app. You can also deploy your Storybook as a static app.
```

Comment:
Rather than making a separate note, would be nice to integrate organically into the paragraph as a first sentence. It doesn't need to be super thorough. Just saying "React Storybook is a project that <...>" makes it clear enough.